src/Fantomas.Tests/FunctionDefinitionTests.fs

```diff
@@ -278,9 +278,9 @@ open System.Runtime.InteropServices
 open Accessibility
 
 [<DllImport("oleacc.dll")>]
-extern int AccessibleChildren(IAccessible paccContainer, int iChildStart, int cChildren, [<Out; MarshalAs(UnmanagedType.LPArray,
```

Comment:
that's a long extern line! I guess this is a bug too, is it filed as an issue?

Reply:
Yes, following up on this in https://github.com/fsprojects/fantomas/issues/735